pysat/instruments/methods/testing.py

```diff
@@ -1,32 +1,40 @@
+import datetime as dt
 import numpy as np
 import os
+import warnings
 
 import pandas as pds
 
-import pysat
 
+ackn_str = ' '.join(("Test instruments provided through the pysat project.",
+                     "https://www.github.com/pysat/pysat"))
+refs = ' '.join(("Russell Stoneback, Jeff Klenzing, Angeline Burrell, Carey",
+                 "Spence, Asher Pembroke, Matthew Depew, … Asher Pembroke.",
+                 "(2019, November 18). pysat/pysat v2.1 (Version v2.1).",
+                 "Zenodo. http://doi.org/10.5281/zenodo.3546270"))
```

Comment:
Where is the other location that reads this citation in? My guess is you are talking about citation.rst but it just has general terms.

I have a nominal citation.txt ready to add. Couple notes. We have to put in the latest DOI, rather than version specific DOI, since it won't exist until after release. I reduced the date to year alone.

Reply:
Yes, that's the location I was thinking of.  I think the citation.txt with the general DOI and only the year is the way to go.  I'd also use Stoneback et al. for the ref, since it's the only bit of the Authors that will remain unchanged across all time.